model/common_enums.smithy

```diff
@@ -65,3 +65,26 @@ string SearchType
      name: "always"}
 ])
 string SuggestMode
+
+@enum([
+    {
+     value: "green",
+     name: "green"},
+     {
+     value: "yellow",
+     name: "yellow"},
+    {
+     value: "red",
+     name: "red"}
+])
+string HealthStatus
+
+@enum([
+    {
+     value: "Accurate",
+     name: "eq"},
+     {
+     value: "Lower bound including returned documents",
```

Comment:
pls confirm if this is the correct "value" for "gte" enum? looks like this might be the description of the enum

Reply:
There was a mistake here. Correcting this.